### DISCUSSION TITLE
Guide: How to use Unchained on Windows 11 (through WSA) to add magnets and torrents to RD.

### DISCUSSION THREAD

**Discussion by morpheasgr** [LivingWithHippos/unchained-android#187] [Ideas]
This guide requires setting up Windows Subsystem for Android on Windows 11 and requires to sideload Aurora Store (or Google Play Store through rooting).

Recommended read:
https://www.reddit.com/r/Windows11/comments/qcqcyd/windows_subsystem_for_android_wsa_megathread/

Aurora Store:
https://files.auroraoss.com/AuroraStore/Stable/

Magnetizer:
https://www.softpedia.com/get/Internet/File-Sharing/Magnetizer.shtml

As soon as you have installed Unchained it will appear as a handler for magnet links. Sadly there is no way to associate file types with Android apps on Windows 11/WSA yet afaik, so I have found a workaround that requires one additional app called Magnetizer, which can be set to open torrent files and can convert them to magnet URIs to be passed on to Unchained. You may then download the contents added to your debrid account outside of the Android environment, by heading to https://real-debrid.com/torrents

If you wish to also handle filehost downloads (if not already possible) this project might be worth a look at: https://github.com/efraimbart/OpenInWSA
But that can also be done with other software such as jDownloader2.

**Comment by Invictaz** [comment]
Why is Aurora needed, there is a Unchained .apk file available?

**Reply by morpheasgr** [reply]
Does the Github version have an updater? In case not, that is why I recommended getting it through Aurora Store, because it keeps apps up to date. In case it does I'll edit the post.

**Reply by LivingWithHippos** [reply]
Thanks for the post, when I have the time I will test it!

**Reply by Invictaz** [reply]
@morpheasgr The Aurora version has less plugins than the Github version. So it's a crossroad. Either you get more updates or get more plugins
